fhir-server-test/src/test/java/com/ibm/fhir/cli/test/FHIRCliTest.java

```diff
@@ -208,17 +208,25 @@ public void testConditionalDeletePatientError() throws Exception {
     
     @Test(dependsOnMethods={"testCreatePatient"})
     public void testConditionalCreatePatient() throws Exception {
-        assertNotNull(patientId);
         runTest("testConditionalCreatePatient", "-p", propsFile(), "--operation", "conditional-create", "--resource", testData("Patient_MookieBetts.json"), "-qp", "_id=" + patientId);
-        // According to https://www.hl7.org/fhir/R4/http.html#create , condition create against an existing resource should return 200
+        // According to https://www.hl7.org/fhir/R4/http.html#ccreate , conditional create against an existing resource should return 200
```

Comment:
is the extra c in ccreate intentional?

Reply:
yes.  now it links to 'conditional create' and not just 'create'